README.md

```diff
@@ -22,6 +22,8 @@ This addon is designed to add the XHGui service to a project served by DDEV.
 This addon is intended for debugging in a development environment.
 Profiling in a production environment is not recommend.
 
+It's therefore wise to add `xhgui.collector.php` and `xhgui.collector.config.php` to your `.gitignore` so it won't accidentally deploy to your production website.
```

Comment:
See my review in https://github.com/tyler36/ddev-xhgui/pull/5 - we don't even need to copy those in, we can use them directly. So this suggestion would be obsolete.

Reply:
Good call, I'll close this PR.